src/flint/fmpz.jl

```diff
@@ -475,6 +475,10 @@ function cdiv(x::fmpz, c::Int)
     return z
 end
 
+rem(x::Integer, y::fmpz) = rem(rem(x), y)
```

Comment:
Are you sure you want `rem(rem(x), y)` here and not `rem(fmpz(x), y)`?

Reply:
Ah sorry, my bad.